frontend_lib/src/css/Variable.styl

```diff
@@ -29,7 +29,7 @@ listItemHeight = 5px
 popupWidth = 420px
 bigIconSize = 32px
 previewIconFontSize = 30px
-
+htmlContentFontSize = fontSizeSmall
```

Comment:
Why don't you just use variable `fontSizeSmall`?

Reply:
Because I want to specify what / the meaning, not the result.

This variable means: This font size is appropriate for html content. Not: we want small font size. If one day we decide that hml contents should have a normal font size, we can just change this line without affecting other things in Tracim that have a small font size.

Maybe the name should even be more general, but I don't know yet what other thing should be concerned by this variable.

We should tend towards this in the frontend I think. I can move this variable in the html content component however.

I think the same thing should be applied to offWhite, lightGrey and the likes: we should identify to what they apply, and use a meaningful name accordingly. It should not matter whether the color is actually gray or offwhite. But we don't know this yet, so we can do this just now.

Reply:
My feeling with the names of all these constants is that they look like they are named after their values, not their meaning.

Reply:
If it is specific to htmlContent, I think it should be in the htmlContent stylus file.